apps/modernization-ui/src/apps/search/patient/result/patientSearchResult.tsx

```diff
@@ -3,11 +3,14 @@ import { displayName } from 'name';
 import { displayAddress } from 'address/display';
 import { Link } from 'react-router-dom';
 
-const displayProfileLink = (result: PatientSearchResult) => (
-    <Link to={`/patient-profile/${result.shortId}/summary`}>
-        {(result.legalName && displayName('fullLastFirst')(result.legalName)) || 'No Data'}
-    </Link>
-);
+const displayProfileLink = (shortId: number, displayName?: string) => {
```

Comment:
just curious: Does changing the signature break anything, not caught by tests? Would an overload with `(result: PatientSearchResult)` for backwards compat be appropriate here?

Reply:
Yeah and the shortId is always provided when calling this function, so there's no risk of it being undefined.

Reply:
Okay no problem. I just had something like this in mind that would keep it consistent with the other "display" functions:
```
const displayProfileLink = ({ shortId }: PatientSearchResult, displayName?: string) => {
```